src/Infrastructure/Adapter/IgnReportClient.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Infrastructure\Adapter;
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\DependencyInjection\Attribute\Autowire;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+use Symfony\Contracts\HttpClient\ResponseInterface;
+
+final class IgnReportClient
+{
+    public function __construct(
+        #[Autowire(service: 'ign.report.client')]
+        private HttpClientInterface $ignReportClient,
+        private LoggerInterface $logger,
+        #[Autowire(env: 'API_IGN_REPORT_AUTH')]
+        private string $credentials,
+        #[Autowire(env: 'IGN_REPORT_STATUS')]
+        private string $defaultStatus = 'test',
```

Comment:
```suggestion
        private string $defaultStatus,
```

Je laisserai cette valeur s'initialiser avec ce qui est défini dans le .env